src/components/PlayerView/index.tsx

```diff
@@ -62,6 +63,10 @@ export interface PlayerViewProps extends BasePlayerViewProps, PlayerViewEvents {
    * To use this property, a `FullscreenHandler` must be set.
    */
   isFullscreenRequested?: Boolean;
+  /**
+   * The `Scaling Mode` that is used by the `PlayerView` to control the scaling mode.
```

Comment:
Please use the documentation similar to the iOS counterpart: https://cdn.bitmovin.com/player/ios/3/docs/Protocols/UserInterfaceApi.html#/c:@M@BitmovinPlayerCore@objc(pl)BMPUserInterfaceApi(py)scalingMode

```suggestion
   * A value defining how the video is displayed within the parent container's bounds.
   * Possible values are defined in `ScalingMode`.
```